database/seed.js

```diff
@@ -74,32 +80,34 @@ var brands = [
   'The North Face'
 ];
 
-var seedData = [];
+const seedData = [];
 
-var returnRandomIndex = (length) => {
+const returnRandomIndex = (length) => {
   return Math.floor(Math.random() * length);
 }
 
-var buildSeedData = (productTypes) => {
+const buildSeedData = (productTypes) => {
 
-  var makeItemEntries = (item, itemUrls) => {
-    for (var i = 0; i < itemUrls.length; i++) {
-      var entry = {};
-      // var nameExtension = '';
+  const makeItemEntries = (item, itemUrls) => {
+    for (let i = 0; i < itemUrls.length; i++) {
+      let entry = {};
 
```

Comment:
Can you assign variables without reference type?

Reply:
Good catch! I updated my seed file, and sent an updated pull request.